api/layer/layer.go

```diff
@@ -569,6 +569,10 @@ func (n *layer) deleteObject(ctx context.Context, bkt *data.BucketInfo, settings
 		}
 
 		if obj.DeleteMarkVersion, obj.Error = n.removeOldVersion(ctx, bkt, nodeVersion, obj); obj.Error != nil {
+			if strings.Contains(obj.Error.Error(), "2050 message = object is locked") {
```

Comment:
How is it related to #988?

Reply:
My fault. It is not related. Wrong branch